edward/stats/distributions.py

```diff
@@ -443,7 +688,22 @@ def entropy(self, mu):
 
 class T:
     def rvs(self, df, loc=0, scale=1, size=1):
-        return stats.t.rvs(df, loc=loc, scale=scale, size=size)
+        if not isinstance(df, np.ndarray):
+            df = np.asarray(df)
+        if not isinstance(loc, np.ndarray):
+            loc = np.asarray(loc)
+        if not isinstance(scale, np.ndarray):
+            scale = np.asarray(scale)
+        if len(df.shape) == 0:
+            return stats.t.rvs(df, loc=loc, scale=scale, size=size)
```

Comment:
i added doc to explain this. basically, the parameters must have consistent shapes/types